internal/pubsub/pubsub.go

```diff
@@ -244,19 +241,18 @@ func (s *Subscription) Close() error {
 // It is for use by provider implementations.
 func NewSubscription(ctx context.Context, d driver.Subscription, opts *SubscriptionOptions) *Subscription {
 	handler := func(item interface{}) {
-		mecs := item.([]msgErrChan)
+		ms := item.([]*Message)
 		var ids []driver.AckID
-		for _, mec := range mecs {
-			m := mec.msg
+		for _, m := range ms {
 			id := m.ackID
 			ids = append(ids, id)
 		}
 		err := d.SendAcks(ctx, ids)
-		for _, mec := range mecs {
-			mec.errChan <- err
+		if err != nil {
```

Comment:
Definitely do not want to panic here. 

Maybe `driver.SendAcks` should not return an error anymore, if we're not going to use it for anything....? Or maybe the concrete type should retry...?